docs/DeveloperGuide.md

```diff
@@ -27,6 +27,40 @@ The application comprises the following components:
 The above *Sequence Diagram* shows how the different components of the system interact with one
 another in the scenario when the command `add -e event -t 1200 -v venue` is executed.
 
+
+## Implementation
+
+This section describes some noteworthy details on how certain features are implemented.
+
+### list feature[TBD]
```

Comment:
I think the "l" in "list" can be capitalised?

Reply:
updated，and I will merge it.